src/Commands/CommandFactory.cs

```diff
@@ -161,13 +161,15 @@ private void RegisterMonitorCommands()
         var monitorTable = new CommandGroup("table", "Log Analytics workspace table operations - Commands for listing tables in Log Analytics workspaces.");
         monitor.AddSubGroup(monitorTable);
 
+        var monitorTableType = new CommandGroup("table-type", "Log Analytics workspace table type operations - Commands for listing table types in Log Analytics workspaces.");
+        monitor.AddSubGroup(monitorTableType);
+
         // Register Monitor commands
-        logs.AddCommand("query", new Monitor.Log.LogQueryCommand(
-            GetLogger<Monitor.Log.LogQueryCommand>()));
-        workspaces.AddCommand("list", new Monitor.Workspace.WorkspaceListCommand(
-            GetLogger<Monitor.Workspace.WorkspaceListCommand>()));
-        monitorTable.AddCommand("list", new Monitor.Table.TableListCommand(
-            GetLogger<Monitor.Table.TableListCommand>()));
+        logs.AddCommand("query", new Monitor.Log.LogQueryCommand(GetLogger<Monitor.Log.LogQueryCommand>()));
```

Comment:
Can you also add the "show" command?  Maybe as a future follow up? 

- list = list all table types
- query = filter table types
- show = get details for a specific table type